homeassistant/components/zwave_js/device_automation_helpers.py

```diff
@@ -41,3 +47,21 @@ def generate_config_parameter_subtype(config_value: ConfigurationValue) -> str:
         parameter = f"{parameter}[{hex(config_value.property_key)}]"
 
     return f"{parameter} ({config_value.property_name})"
+
+
+@callback
+def async_bypass_dynamic_config_validation(hass: HomeAssistant, device_id: str) -> bool:
```

Comment:
Should we break out this change to a separate PR as it's a fix? We may want to include it in a patch release, but the api changes are not for a patch release, right?

Reply:
yes we can do that. The main PR here is technically a bug fix too for https://github.com/home-assistant/frontend/issues/12641 and https://github.com/home-assistant/frontend/issues/12136 but it's a large change so maybe we want to wait until the next release?

Reply:
Yeah, I'm not sure that we can or should include both this and the needed frontend PR in a patch release.